test/verify/check-realms

```diff
@@ -236,7 +236,7 @@ class TestRealms(MachineCase):
         b.click("#system-info-domain a")
         b.wait_popup("realms-op")
         b.set_val(".realms-op-address", "NOPE")
-        b.wait_js_cond("ph_find('.realms-op-address-error').title != ''")
+        b.wait_visible(".realms-op-address-error")
```

Comment:
Can we please check the error message content here as well? The tooltip should say something like "The domain NOPE could not be contacted". `git grep mouseover test/verify/` shows existing tests that validate tooltips.

Reply:
Tried that but
``` python
b.mouse(".realms-op-address-error span", "mouseover")
```
doesn't trigger the tooltip. Any idea why?

Reply:
It most likely catches the wrong element which doesn't have the mouseover handler. Try to drop the `span`? Otherwise, check in the inspector which particular element has the handler.

Reply:
I still don't get how it gets triggered without handler on any element. Thanks a lot anyway!!